lib/buildkite/pipelines/steps/block.rb

```diff
@@ -16,6 +16,7 @@ class Block < Abstract
         attribute :branches
         attribute :fields
         attribute :blocked_state
+        attribute :allowed_teams
```

Comment:
This might make sense as an appended attribute.

```suggestion
        attribute :allowed_teams, append: true
```

Reply:
Sounds good. Change has been made now.